content/docs/faq-functions.md

```diff
@@ -149,28 +164,33 @@ Alternately, you can use DOM APIs to store data needed for event handlers. Consi
 
 ```jsx
 const A = 65 // ASCII character code
+
 class Alphabet extends React.Component {
-  state = {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+    this.state = {
       justClicked: null,
       letters: Array.from({length: 26}, (_, i) => String.fromCharCode(A + i))
+    };
   }
-  
-  handleClick = event => {
+
+  handleClick(e) {
```

Comment:
Intentionally using `e` in these examples. When it's typed as `event` it's easy to forget the `event` argument and accidentally use the `window.event` global that is provided by the browser for IE compatibility (and has nothing to do with React).